habitat/utils/gym_adapter.py

```diff
@@ -294,17 +301,37 @@ def reset(self) -> Union[np.ndarray, Dict[str, np.ndarray]]:
         self._last_obs = obs
         return self._transform_obs(obs)
 
-    def render(self, mode: str = "rgb_array") -> np.ndarray:
+    def render(self, mode: str = "human") -> np.ndarray:
         frame = None
         if mode == "rgb_array":
             frame = observations_to_image(
                 self._last_obs, self._env._env.get_metrics()
             )
+        elif mode == "human":
+            if pygame is None:
+                raise ValueError(
+                    "Render mode human not supported without pygame."
+                )
+            frame = observations_to_image(
+                self._last_obs, self._env._env.get_metrics()
+            )
+            if self._screen is None:
+                pygame.init()
+                self._screen = pygame.display.set_mode(
+                    [frame.shape[1], frame.shape[0]]
+                )
+            draw_frame = np.transpose(frame, (1, 0, 2))
+            draw_frame = pygame.surfarray.make_surface(draw_frame)
+            self._screen.fill((0, 0, 0))  # type: ignore[attr-defined]
```

Comment:
Let's avoid magic unnamed numerical consts:
```suggestion
            BLACK_COLOR = (0, 0, 0)
            self._screen.fill(BLACK_COLOR)  # type: ignore[attr-defined]
```